cnd/src/btsieve/ethereum/mod.rs

```diff
@@ -243,9 +243,9 @@ async fn process_next_find_parent<C, E>(
     }
 }
 
-/// Process hashes from the `next_look_in_the_past` receiver.  Gets the block
+/// Process hashes from the `next_look_in_the_past` receiver. Gets the block
```

Comment:
No need to change but I always use two spaces after a full stop.  Researching this for this post shows that I'm wrong - will now switch to using one space.  FTR my emacs setup does not do 're column width' i.e., update region to use the correct column width, correctly when there is only one space (that's why I started using two).